app/lib/activitypub/activity/follow.rb

```diff
@@ -6,14 +6,14 @@ def perform
 
     return if target_account.nil? || !target_account.local? || delete_arrived_first?(@json['id']) || @account.requested?(target_account)
 
+    follow_request = FollowRequest.create!(account: @account, target_account: target_account, remote_id: @json['id'])
```

Comment:
I do not think writing an object which lives just in an execution context to persistent database can be tolerated. What about passing `FollowRequest` object to `AuthorizeFollowService` _without_ making it persistent?